openmc/plotter.py

```diff
@@ -0,0 +1,588 @@
+        4.95 weight percent enriched U. Default is None
+        (natural composition).
+
+    Returns
+    -------
+    energy_grid : numpy.ndarray
+        Energies at which cross sections are calculated, in units of eV
+    data : numpy.ndarray
+        Cross sections calculated at the energy grid described by energy_grid
+
+    """
+
+    # Check types
+    cv.check_type('temperature', temperature, Real)
+    if sab_name:
+        cv.check_type('sab_name', sab_name, str)
+    if enrichment:
+        cv.check_type('enrichment', enrichment, Real)
+
+    if isinstance(this, openmc.Nuclide):
+        energy_grid, data = _calculate_xs_nuclide(this, types, temperature,
+                                                  sab_name, cross_sections)
+    elif isinstance(this, openmc.Element):
+        energy_grid, data = _calculate_xs_element(this, types, temperature,
+                                                  sab_name, cross_sections,
+                                                  enrichment)
+    elif isinstance(this, openmc.Material):
+        energy_grid, data = _calculate_xs_material(this, types, temperature,
+                                                   cross_sections)
+    else:
+        raise TypeError("Invalid type")
+
+    return energy_grid, data
+
+
+def _calculate_xs_element(this, types, temperature=294., sab_name=None,
+                          cross_sections=None, enrichment=None):
+    """Calculates continuous-energy cross sections of a requested type
+
+    Parameters
+    ----------
+    this : openmc.Element
+        Element object to source data from
+    types : Iterable of values of PLOT_TYPES
+        The type of cross sections to calculate
+    temperature : float, optional
+        Temperature in Kelvin to plot. If not specified, a default
+        temperature of 294K will be plotted. Note that the nearest
+        temperature in the library for each nuclide will be used as opposed
+        to using any interpolation.
+    sab_name : str, optional
+        Name of S(a,b) library to apply to MT=2 data when applicable.
+    cross_sections : str, optional
+        Location of cross_sections.xml file. Default is None.
+    enrichment : float, optional
+        Enrichment for U235 in weight percent. For example, input 4.95 for
+        4.95 weight percent enriched U. Default is None
+        (natural composition).
+
+    Returns
+    -------
+    energy_grid : numpy.ndarray
+        Energies at which cross sections are calculated, in units of eV
+    data : numpy.ndarray
+        Macroscopic cross sections calculated at the energy grid described
+        by energy_grid
+
+    """
+
+    # Load the library
+    library = openmc.data.DataLibrary.from_xml(cross_sections)
+
+    # Expand elements in to nuclides with atomic densities
+    nuclides = this.expand(100., 'ao', enrichment=enrichment,
+                           cross_sections=cross_sections)
```

Comment:
`100.` should be changed to `1.`. Otherwise, the element cross section will be 100x too big!

![zr](https://cloud.githubusercontent.com/assets/1244537/20400689/c3a7c9a4-acc3-11e6-8f33-3898d1df0e4c.png)